appeal/job_handler.go

```diff
@@ -57,3 +65,40 @@ func (h *JobHandler) RevokeExpiredAccess() error {
 	log.Println(string(result))
 	return nil
 }
+
+func (h *JobHandler) NotifyAboutToExpireAccess() error {
+	daysBeforeExpired := []int{7, 3, 1}
+	for _, d := range daysBeforeExpired {
+		now := time.Now().AddDate(0, 0, d)
+		year, month, day := now.Date()
+		from := time.Date(year, month, day, 0, 0, 0, 0, now.Location())
+		to := time.Date(year, month, day, 23, 59, 59, 999999999, now.Location())
+
+		filters := map[string]interface{}{
+			"statuses":           []string{domain.AppealStatusActive},
+			"expiration_date_gt": from,
+			"expiration_date_lt": to,
+		}
+
+		appeals, err := h.appealService.Find(filters)
+		if err != nil {
+			h.logger.Error(err.Error())
```

Comment:
should we add more information such as `dayBeforeExpired` (e.g. 7, 3, or 1)?

also if it fails, would it be better to `continue` or else the remaining actions will be run.
e.g. 
```
if err != nil {
	h.logger.Errorf("error finding appeals \"%d\" days before expiration: %s", err.Error())
        continue
}
```